resources/js/components/Checkout/CheckoutSuccess.vue

```diff
@@ -4,12 +4,44 @@
     export default {
         mixins: [GetCart],
 
+        props: {
+            token: {
+                type: String,
+                default: null,
+            },
+            mask: {
+                type: String,
+                default: null
+            }
+        },
+
+        data() {
+            return {
+                order: {}
+            }
+        },
+
         render() {
-            return this.$scopedSlots.default()
+            return this.$scopedSlots.default({
+                order: this.order,
+                refreshOrder: this.refreshOrder,
+            })
         },
 
         created() {
+            this.token ??= localStorage.token;
+            this.mask ??= localStorage.mask;
+
+            this.refreshOrder();
             this.clearCart()
+        },
+
+        methods: {
+            refreshOrder()
+            {
+                axios.get('/api/order/' + (this.token ? this.token : this.mask))
```

Comment:
This also works I think?
```suggestion
                axios.get('/api/order/' + (this.token || this.mask))
```